rpc/jsonrpc/jsonrpc_test.go

```diff
@@ -84,22 +85,24 @@ func TestMain(m *testing.M) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	setup(ctx)
+	if err := setup(ctx); err != nil {
+		stdlog.Fatal(err.Error())
```

Comment:
why the import rename?

Reply:
i believe we use our loging package as the same name :(